File: android/app/src/main/java/com/smartexpense/android/data/remote/dto/response/NotificationResponseDto.java
package com.smartexpense.android.data.remote.dto.response;

public class NotificationResponseDto {
    private String id;
    private String type;
    private String content;
    private boolean isRead;
    private String createdAt;

    public String getId() { return id; }
    public String getType() { return type; }
    public String getContent() { return content; }
    public boolean getIsRead() { return isRead; }
    public String getCreatedAt() { return createdAt; }
}

File: android/app/src/main/kotlin/com/smartexpense/android/data/remote/dto/response/NotificationResponseDto.kt
package com.smartexpense.android.data.remote.dto.response

import com.google.gson.annotations.SerializedName

data class NotificationResponseDto(
    @SerializedName("id") val id: String,
    @SerializedName("type") val type: String,
    @SerializedName("content") val content: String,
    @SerializedName("isRead") val isRead: Boolean,
    @SerializedName("createdAt") val createdAt: String
)
